### DISCUSSION TITLE
Add Test Suites in the Tests View

### DISCUSSION THREAD

**Discussion by AllanOricil** [forcedotcom/salesforcedx-vscode#5350] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently, the tests view displays Individual Apex tests only. I believe it would be a great addition to DX if users could also see tests suites. Tests can be added to existing Apex Test Suites via the `SFDX: Add Tests to Apex Test Suites` command in the Command Palette but there's currently no way to view these within the Test Sidebar. Apex test suites allow users to group Apex tests by subject or context.

![image](https://user-images.githubusercontent.com/55927613/137639264-0634eeb5-db2b-41c8-a18b-90bd4f30e365.png)

**Describe the solution you'd like**
![image](https://user-images.githubusercontent.com/55927613/137639326-305ca6a1-5322-4a0b-9c8d-9c54322acbd3.png)

**Describe alternatives you've considered**

Users have to open a terminal and type an sfdx command.

**Additional context**
N/A

**Comment by benji1304** [comment]
Is there anywhere to view a test suite? Where are they saved?

**Comment by AllanOricil** [comment]
Could you guys see if there is anybody interested in this feature? I can do it in my free time if people want it

**Reply by AnanyaJha** [reply]
Hi @AllanOricil ! I've got the same question as you and would personally love to see this feature added! Converted to discussion so we can hear from more users. Thanks for your engagement, as always!

**Comment by AllanOricil** [comment]
@mshanemc @peternhale maybe you can do this

**Comment by mshanemc** [comment]
@CristiCanizales did it
<img width="456" height="471" alt="Screenshot 2026-04-22 at 9 36 53 PM" src="https://github.com/user-attachments/assets/53913250-4c6f-4dfa-bc76-e8ccbc6a8f0b" />
